src/Umbraco.Cms.StaticAssets/Umbraco.Cms.StaticAssets.csproj

```diff
@@ -4,77 +4,95 @@
+    <CompressionEnabled>false</CompressionEnabled>
   </PropertyGroup>
 
   <PropertyGroup>
-    <!--  TODO: [NU5123] Rename files so path is shorter and remove this override -->
-    <WarningsNotAsErrors>NU5123</WarningsNotAsErrors>
+    <!--
+      TODO: Fix and remove overrides:
+      [NU5123] Rename files so path is shorter
+    -->
+    <WarningsNotAsErrors>$(WarningsNotAsErrors),NU5123</WarningsNotAsErrors>
   </PropertyGroup>
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <SupportedPlatform Include="browser" />
+  </ItemGroup>
+
+  <ItemGroup>
     <ProjectReference Include="..\Umbraco.Cms.Api.Management\Umbraco.Cms.Api.Management.csproj" />
     <ProjectReference Include="..\Umbraco.Web.Website\Umbraco.Web.Website.csproj" />
   </ItemGroup>
 
+  <!-- Restore and build backoffice project -->
   <PropertyGroup>
-    <BasePath>$(ProjectDir)wwwroot\umbraco</BasePath>
-    <BellissimaPath>$(BasePath)\backoffice</BellissimaPath>
-    <LoginPath>$(BasePath)\login</LoginPath>
+    <BackofficeProjectDirectory Condition="'$(BackofficeProjectDirectory)' == ''">..\Umbraco.Web.UI.Client\</BackofficeProjectDirectory>
+    <BackofficeAssetsPath>wwwroot\umbraco\backoffice</BackofficeAssetsPath>
   </PropertyGroup>
 
-  <Target Name="BuildStaticAssetsPreconditions" BeforeTargets="Build">
-    <Message Text="Skip BuildBellissima target because UmbracoBuild is '$(UmbracoBuild)' (this is not Visual Studio)" Importance="high" Condition="'$(UmbracoBuild)' != ''" />
-    <Message Text="Skip BuildBellissima target because '$(BellissimaPath)' already exists" Importance="high" Condition="Exists('$(BellissimaPath)')" />
-    <Message Text="Call BuildBellissima target because UmbracoBuild is empty (this is Visual Studio) and '$(BellissimaPath)' doesn't exist" Importance="high" Condition="'$(UmbracoBuild)' == '' and !Exists('$(BellissimaPath)')" />
-    <CallTarget Targets="BuildBellissima" Condition="'$(UmbracoBuild)' == '' and !Exists('$(BellissimaPath)')" />
-
-    <Message Text="Skip BuildLogin target because UmbracoBuild is '$(UmbracoBuild)' (this is not Visual Studio)" Importance="high" Condition="'$(UmbracoBuild)' != ''" />
-    <Message Text="Skip BuildLogin target because '$(LoginPath)' already exists" Importance="high" Condition="Exists('$(LoginPath)')" />
-    <Message Text="Call BuildLogin target because UmbracoBuild is empty (this is Visual Studio) and '$(LoginPath)' doesn't exist" Importance="high" Condition="'$(UmbracoBuild)' == '' and !Exists('$(LoginPath)')" />
-    <CallTarget Targets="BuildLogin" Condition="'$(UmbracoBuild)' == '' and !Exists('$(LoginPath)')" />
-  </Target>
+  <ItemGroup>
+    <BackofficeAssetsInputs Include="$(BackofficeProjectDirectory)package.json;$(BackofficeProjectDirectory)package-lock.json;$(BackofficeProjectDirectory)src\**" Exclude="$(DefaultItemExcludes)" />
+    <Content Remove="$(BackofficeAssetsPath)\**" />
```

Comment:
I've removed the generated files from the project, so they don't show up and also to avoid duplicate errors due to them otherwise being added to the `StaticWebAsset` item group by default (the custom `DefineBackofficeAssets` task adds them already)...